packages/muelu/doc/UsersGuide/masterList.xml

```diff
@@ -749,6 +748,30 @@
       <comment-ML>parameter not existing in ML</comment-ML>
     </parameter>
 
+    <parameter>
+      <name>aggregate qualities: algorithm</name>
+      <type>string</type>
+      <default>forward</default>
+      <description>Algorithm to use ``forward'' or ``reverse.''</description>
+      <comment-ML>parameter not existing in ML</comment-ML>
+    </parameter>
+    
+    <parameter>
+      <name>aggregate qualities: zero threshold</name>
+      <type>double</type>
+      <default>1e-12</default>
+      <description>The zero threshold for the reverse algorithm.</description>
+      <comment-ML>parameter not existing in ML</comment-ML>
```

Comment:
Do you want these options visible?

Reply:
Well, the aggregate quality stuff is add visible...